editquality/feature_lists/wikidatawiki.py

```diff
@@ -115,8 +162,14 @@ class items:
     is_blp
 ]
 
+comment_related = [
+    comment_longest_repeated_char,
+    comment_uppercase_ratio,
+    comment_numbers_ratio,
+    comment_whitespace_ratio
+]
 damaging = mediawiki.comment + mediawiki.user_rights + \
            mediawiki.protected_user + wikibase.parent + wikibase.diff + \
-           local_wiki
+           local_wiki + comment_related
 reverted = damaging
 goodfaith = damaging
```

Comment:
This changes the features for reverted/damaging/goodfaith models but it only rebuilds the damaging model.